src/features/processPicker.ts

```diff
@@ -35,6 +37,80 @@ export class AttachPicker {
     }
 }
 
+export class RemoteAttachPicker {
+    public static ShowAttachEntries(args : any): Promise<string> {
+        // Grab selected name from UI
+        var name : string = args.name;
+
+        // Build path for launch.json to find pipeTransport
+        const vscodeFolder : string = path.join(vscode.workspace.rootPath, '.vscode');
+        var launchJsonPath : string = path.join(vscodeFolder, 'launch.json')
+
+        // Read launch.json
+        var json : any = JSON.parse(fs.readFileSync(launchJsonPath).toString());
+
+        // Find correct pipeTransport via selected name
+        var config; 
+        for (var i = 0; i < json.configurations.length; ++i) {
+            if (json.configurations[i].name === name) {
+                config = json.configurations[i];
+                break; 
+            }
+        }
+
+        if (!config.pipeTransport) {
+            // Missing PipeTransport, prompt if user wanted to just do local attach?
+            return new Promise<string>((resolve, reject) => {
+                reject(new Error("Configuration \"" + args.name + "\" in launch.json does not have a " + 
+                "pipeTransport argument for pickRemoteProcess. Use pickProcess for local attach."));
+            });
+        } else {
+            var pipeCmd : string = config.pipeTransport.pipeProgram + " " + config.pipeTransport.pipeArgs[0];
```

Comment:
You want to join the arguments together, not just take the 0th entry from the array.